tutorials/shaders/compute_shaders.rst

```diff
@@ -30,7 +30,9 @@ So now lets work with a compute shader to see how it really works.
 Creating a ComputeShader
 ------------------------
 
-To begin using compute shaders, create a new text file called "compute_example.glsl". When you write compute shaders in Godot, you write them in GLSL directly. The Godot shader language is based off of GLSL so if you are familiar with normal shaders in Godot the syntax below will look somewhat familiar.
+To begin using compute shaders, ensure you are using a Vulkan based renderer (the Forward+ or Mobile options), or your editor may crash.
```

Comment:
I wouldnt mention crashing. If the editor crashes instead of printing an error then it is a bug that needs to be fixed, not a feature that needs to be documented.